website/docs/r/event_orchestration_service.html.markdown

```diff
@@ -0,0 +1,202 @@
+alert_creation          = "create_alerts_and_incidents"
+}
+
+resource "pagerduty_event_orchestration_service" "www" {
+  service = pagerduty_service.example.id
+  sets {
+    id = "start"
+    rules {
+      label = "Always apply some consistent event transformations to all events"
+      actions {
+        variables {
+          name = "hostname"
+          path = "event.component"
+          value = "hostname: (.*)"
+          type = "regex"
+        }
+        extractions {
+          # Demonstrating a template-style extraction
+          template = "{{variables.hostname}}"
+          target = "event.custom_details.hostname"
+        }
+        extractions {
+          # Demonstrating a regex-style extractions
+          source = "event.source"
+          regex = "www (.*) service"
+          target = "event.source"
+        }
+        # Id of the next set
+        route_to = "step-two"
+      }
+    }
+  }
+  sets {
+    id = "step-two"
+    rules {
+      label = "All critical alerts should be treated as P1 incident"
+      conditions {
+        expression = "event.severity matches 'critical'"
+      }
+      actions {
+        annotate = "Please use our P1 runbook: https://docs.test/p1-runbook"
+        priority = "P0IN2KQ"
+      }
+    }
+    rules {
+      label = "If there's something wrong on the canary let the team know about it in our deployments Slack channel"
+      conditions {
+        expression = "event.custom_details.hostname matches part 'canary'"
+      }
+      # create webhook action with parameters and headers
+      actions {
+        automation_actions {
+          name = "Canary Slack Notification"
+          url = "https://our-slack-listerner.test/canary-notification"
+          auto_send = true
+          parameters {
+            key = "channel"
+            value = "#my-team-channel"
+          }
+          parameters {
+            key = "message"
+            value = "something is wrong with the canary deployment"
+          }
+          headers {
+            key = "X-Notification-Source"
+            value = "PagerDuty Incident Webhook"
+          }
+        }
+      }
+    }
+    rules {
+      label = "Never bother the on-call for info-level events outside of work hours"
+      conditions {
+        expression = "event.severity matches 'info' and not (now in Mon,Tue,Wed,Thu,Fri 09:00:00 to 17:00:00 America/Los_Angeles)"
+      }
+      actions {
+        suppress = true
+      }
+    }
+  }
+  catch_all {
+    actions { }
+  }
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `service` - (Required) ID of the Service to which this Service Orchestration belongs to.
+* `sets` - (Required) A Service Orchestration must contain at least a "start" set, but can contain any number of additional sets that are routed to by other rules to form a directional graph.
+* `catch_all` - (Required) When none of the rules match an event, the event will be routed according to the catch_all settings.
```

Comment:
**Blocker:** This is a Service Orchestration the `catch_all` cannot do routing. Maybe this instead?

```suggestion
* `catch_all` - (Required) the `catch_all` actions will be applied if an Event reaches the end of any set without matching any rules in that set.
```